### PR TITLE
Added example page for Channel Value

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3230

#### Changes proposed in this Pull Request:
- Added examples for Channel value

#### Screenshots:
Screenshot 2022-09-23 at 4 42 53 PM

#### To Test:
- yarn start:reactdev

### CODE REVIEW

**Comment by jeffvg** [comment]
looks good. For the Channel Value with Different Units we mention ```unitSpace``` prop.
Should one of the examples here have unitSpace set?

image

File: docs/src/componentDocs/ChannelValue/examples/ChannelValueWithDifferentUnits.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { CodeBlock, CodeBlockActionButtonRow } from '../../../shared';
+import * as colors from '@brightlayer-ui/colors';
+import { ChannelValueWithDifferentUnitsExample } from './ChannelValueWithDifferentUnitsExample';
+
+const codeSnippet = `<Box sx={{ display: 'flex', flexDirection: 'column', '& .BluiChannelValue-root': { my: 0.25 } }}>
+    <ChannelValue value="85" units="kWh" />
```
Comment (huayunh) [inline docs/src/componentDocs/ChannelValue/examples/ChannelValueWithDifferentUnits.tsx:8]:
```suggestion
    <ChannelValue value="85" units="kWh" />
    <ChannelValue value="85" units="kWh" unitSpace={'show'} />
```

File: docs/src/componentDocs/ChannelValue/examples/ChannelValueWithDifferentUnitsExample.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import { ChannelValue } from '@brightlayer-ui/react-components';
+
+export const ChannelValueWithDifferentUnitsExample = (): JSX.Element => (
+    <Box sx={{ display: 'flex', flexDirection: 'column', '& .BluiChannelValue-root': { my: 0.25 } }}>
+        <ChannelValue value="85" units="kWh" />
```
Comment (huayunh) [inline docs/src/componentDocs/ChannelValue/examples/ChannelValueWithDifferentUnitsExample.tsx:7]:
```suggestion
        <ChannelValue value="85" units="kWh" />
        <ChannelValue value="85" units="kWh" unitSpace={'show'} />
```

**Review by huayunh — COMMENTED** [review]
Yeah I think it is a good idea. @emclaug2 may want to do something similar on the ng side too?

Changes look good otherwise.

File: docs/src/componentDocs/ChannelValue/examples/ChannelValueWithDifferentUnits.tsx (hunk 1 above)
Comment (bkarambe) [reply docs/src/componentDocs/ChannelValue/examples/ChannelValueWithDifferentUnits.tsx]: Added examples using unitSpace hide and show

**Review by huayunh — APPROVED** [review]

**Pull request merged**